cmd/internal/generator/types.go

```diff
@@ -31,9 +31,16 @@ func (l Label) GoType() string {
 	}
 }
 
+func (l Label) GoFieldValues() string {
```

Comment:
can we make it take "argument name" rather than hardcoding (`v`) it, it might create bugs

Reply:
Fixed. Let me know if it doesn't work for you.